db/migrate/20150710211530_change_events_exception_to_text.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeEventsExceptionToText < ActiveRecord::Migration
+  def up
+    change_column :events, :exception, :text
```

Comment:
@dchandekstark Consider adding `, limit: 65535` (or some such number) to this line.  Otherwise, in certain contexts at least, the limit may end up set too low in the resulting schema.  See, for example, line 28 in spec/dummy/db/schema,rb below, where it is set to 255.